russian/thinking-about/pen-paper/pen-paper.md

```diff
@@ -33,4 +33,4 @@
 Там ты увидел диван, скатерть-самобранку, плазму, коробокс и подписку на Netflix на 100 лет.
 
 
-[...](../../zefir.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.